blackbox/docs/sql/ddl/basics.txt

```diff
@@ -120,6 +120,37 @@ length. They:
 
 Column names are restricted in terms of characters. They:
 
-  - may not contain one of the following characters: ``[`` ``'`` ``]`` ``.``
+  - may not *start* with an underscore: ``_`` followed by lowercase
+    letters (a-z) and underscores while ending with a letter.
+    Such fields are reserved for virtual system columns.
 
-  - may not *start* with an underscore: ``_``
+    Examples:::
+
+    +--------------+--------------+
+    | allowed      | not allowed  |
+    +--------------+--------------+
+    | _foo_        | _foo         |
+    +--------------+--------------+
+    | _Foo         | _foo_bar     |
+    +--------------+--------------+
+    | _fo1o        |              |
+    +--------------+--------------+
+    | __foo        |              |
+    +--------------+--------------+
+
+  - may not start with any character followed by opening and closing square
```

Comment:
A character sequence ...

* ... followed by an opening square bracket ...
* ... followed by zero or more characters which are not closing square brackets ...
* ... followed by a closing square bracket

is not allowed as column name.